src/index.html

```diff
@@ -22,6 +22,346 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="reviews__count">Reviews: 5</div>
+          </div>
+          <div class="price card__price">
+            <p class="price__title">Price:</p>
+            <div class="price__current">$2,199</div>
+          </div>
+          <a
+            class="btn-link btn-link--hover"
+            data-qa="card-hover"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="picture product"
+          />
+          <div class="product card__prod">
+            <h3 class="product__title">
+              APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h3>
+            <p class="product__code">Product code: 195434</p>
+          </div>
+          <div class="reviews card__review">
+            <!-- блок stars -->
+            <div class="stars stars--4 reviews__stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="reviews__count">Reviews: 5</div>
+          </div>
+          <div class="price card__price">
+            <p class="price__title">Price:</p>
+            <div class="price__current">$2,199</div>
+          </div>
+          <a
+            class="btn-link btn-link--hover"
+            data-qa="hover"
+            href="#"
+          >
+            Buy
+          </a>
+        </div>
+        <div class="card">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="picture product"
+          />
+          <div class="product card__prod">
+            <h3 class="product__title">
+              APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h3>
+            <p class="product__code">Product code: 195434</p>
+          </div>
+          <div class="reviews card__review">
+            <!-- блок stars -->
+            <div class="stars stars--4 reviews__stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="reviews__count">Reviews: 5</div>
+          </div>
+          <div class="price card__price">
+            <p class="price__title">Price:</p>
+            <div class="price__current">$2,199</div>
+          </div>
+          <a
+            class="btn-link btn-link--hover"
+            data-qa="hover"
```

Comment:
The `data-qa` attribute is used here as `hover`, but in other similar elements, it is `card-hover`. Ensure consistency in naming conventions for data attributes to avoid confusion.